examples/standalone/README.md

```diff
@@ -141,6 +141,20 @@ Install the following dependencies:
 - [Terraform](https://www.terraform.io/downloads.html) version 0.13.7 or later.
 - [curl](https://curl.haxx.se/) version 7.68.0 or later.
 
+### Cloud SDK configurations
+
+Acquire your user credentials to use for **Application Default Credentials**:
+
+```sh
+gcloud auth application-default login
+```
+
+Unset the project in the core section to avoid errors in the deploy with deleted or unavailable projects, and billings on inappropriate projects:
```

Comment:
interesting...I thought we can set a default billing project in one of the top level configs...was it the provider.tf or something?

Reply:
The billing project set in the provider only has effects in the things build in the terraform code. 

We are recommending the user to unset the project that is on the `gcloud`, because this is the configuration that the `wrapped_key.py` python script uses.